pkg/controller/scale_down.go

```diff
@@ -12,6 +12,12 @@ import (
 	v1 "k8s.io/api/core/v1"
 )
 
+const (
+	// NodeEscalatorIgnoreAnnotation is the key of an annotation on a node that signifies it should be ignored from ASG deletion
+	// value does not matter, can be used for reason, as long as not empty
+	NodeEscalatorIgnoreAnnotation = "com.atlassian/escalator-ignore"
```

Comment:
Why `com.atlassian`? We've used `atlassian.com` in the past in this repo. 

I think we should also signify in the annotation and the comment that the annotation only works for deletion, unlike cordoning. One might think this factors a node out of capacity calculations.

Reply:
changed to `atlassian.com/no-delete` and added comment